### DISCUSSION TITLE
how does they work

### DISCUSSION THREAD

**Discussion by klapvogn** [ryelogheat/xpbot#50] [General]
How would the below work

if i have : 
`7500.2019.NORDiC.REMUX.1080p.BluRay.AVC.DTS-HD.MA.5.1`

the site sets it as : 

Type: Full Disc
Res: 1080p (which is true)

i would guess it should get REMUX in type

`    "type_id": {
      "1": {
        "disc": 1,
        "bluray_disc": 2,
        "dvd_disc": 2
      },
      "2": {
        "remux": 1,
        "bluray_remux": 2,
        "dvd_remux": 2
      },
      "3": {
        "bluray_encode": 1
      },
      "4": {
        "webdl": 1
      },
      "5": {
        "webrip": 1
      },
      "6": {
        "hdtv": 1
      }
    },
`

**Comment by HDVinnie** [comment]
What site?

**Reply by klapvogn** [reply]
danishbytes.club

**Reply by HDVinnie** [reply]
make sure you have proper type_id set in json file. Whats your file look like?

**Comment by klapvogn** [comment]


**Reply by HDVinnie** [reply]
Doesn't tell me much....would need to see your json file so far. Pastebin or alike.

**Comment by klapvogn** [comment]
https://zerobin.net/?9594917ebb0ccc3b#9Kj5GaetIN9/8tP61E8EEayrIBeJEuD9A5qrwWHgKOw=

**Reply by HDVinnie** [reply]
```      
2": {
    "remux": 1,
    "bluray_remux": 2,
    "dvd_remux": 2
  },
```

**Reply by klapvogn** [reply]
Thanks i'll give it a go :-)

**Reply by klapvogn** [reply]
Still shows up as Full Disc
https://prnt.sc/LDlYdrPhH3RI

from log output
`'sd': '0', 'stream': '0', 'tvdb': '0', 'tmdb': '646150', 'imdb': '9534808', 'mal': '0', 'igdb': '0', 'internal': '0', 'free': '0', 'featured': '0', 'doubleup': '0', 'sticky': '0', 'category_id': '1', 'type_id': '1', 'resolution_id': '3',`

**Reply by klapvogn** [reply]
Wonder if it default to `Full Disc`

![dby_2](https://user-images.githubusercontent.com/2506366/155322455-faf28782-7346-426c-80b4-3cbaea9f80f7.png)

**Reply by ryelogheat** [reply]
I'm still not sure if the problem is with your **tracker.json** file or if **auto_upload.py** is misidentifying your file. We need more details, could you open **upload_script.log** and paste everything related with the ` 7500.2019.NORDiC.REMUX.1080p.BluRay.AVC.DTS-HD.MA.5.1` upload (remove any sensitive info like API key or full file paths, etc.)

If you don't want to paste it in this thread then you can just email it to me at ryelogheat@pm.me

I suspect that the problem is with your **tracker.json** config since the python module [guessit](https://pypi.org/project/guessit/) (main way of identifying details about an upload) does correctly identify your file as a Bluray Remux. Still I'm curious to see what the logs say so post them below or email them to me

**Comment by klapvogn** [comment]
Log output:
https://zerobin.net/?a82e4bf0569faa41#sjDf6FvmYdDCIuut5QpxBar8aLigo5CIfoUYYwyafE4=

x.json
https://zerobin.net/?6ad5aa925f70fec3#NVQvtRp4g4AYKgHCfCof0ESmx7rPpLdsF7OjRyQuPxk=
